core/src/main/scala/org/apache/spark/executor/TaskMetrics.scala

```diff
@@ -202,7 +203,7 @@ class TaskMetrics private[spark] () extends Serializable {
   }
 
   // Only used for test
-  private[spark] val testAccum = sys.props.get("spark.testing").map(_ => new LongAccumulator)
+  private[spark] val testAccum = sys.props.get(IS_TESTING.key).map(_ => new LongAccumulator)
```

Comment:
Same.

Reply:
I agree it is doable but the code would become an if which seems to me less clean than the current code